### ISSUE TITLE
I'm unable to open any of my Workspaces;  they can't get past the "Pulling Docker Images...(2/3)    https://b5ba9c52-2ffb-4757-afc2-93ccd69c2f47.ws-us02.gitpod.io/

### ISSUE THREAD

**Issue by BobHerold** [gitpod-io/gitpod#1266]

**Comment by BobHerold** [comment]
they finally loaded after about 15 minutes !

**Comment by ColbyWTaylor** [comment]
Experiencing same issue- successfully opened gitpod, tried doing an npm install and immediately unresponsive (but no "connection warnings" or status update as usual). Eventually went to dashboard, saw it was running, started again, and got caught on the "pulling doccker images 2/3" warning.

Rference space: d13a88b9-8ef1-4ad1-a5a2-d23b10b9d661

**Comment by ColbyWTaylor** [comment]
Just checked again, space opened, but no access to terminal and consistently getting "offline" orange...

**Comment by svenefftinge** [comment]
Thanks for reporting. There were some heavy processes yesterday that burned a lot of CPU on some nodes. These are usually crypto mining or similar. So other workspaces were affected. 

We have been aware of this noisy neighbourhood effect for longer and have finally deployed a fix, which is now active. The change gives each workspace a limited CPU bucket, that will downgrade the number of CPU cores one can use the longer you burn CPUs. 

We have tested this in staging and believe we've found a good setting that will make Gitpod less attractive for CPU-burning tasks, so our good users can do what Gitpod is intended to be used for.

I am closing this but would like to encourage you to file a new ticket if you see this happening again.
